scripts/e2etests/VSUtils.ps1

```diff
@@ -215,6 +215,19 @@ function ResumeVSInstall {
         [int]$ProcessExitTimeoutInSeconds
     )
 
+    # Ensure we have the latest VS_Installer
+    
+    $VSBootstrapperUrl = "https://aka.ms/vs/16/intpreview/vs_enterprise.exe"
+    $tempdir = [System.IO.Path]::GetTempPath()
+    $VSBootstrapperPath =  "$tempdir" + "vs_enterprise.exe"
```

Comment:
In the offline discussion, Andy found (https://docs.microsoft.com/en-us/visualstudio/install/use-command-line-parameters-to-install-visual-studio?view=vs-2019#command-line-parameters), 
`Replace vs_enterprise.exe as appropriate for the product edition you're installing. (Alternatively, you can use vs_installer.exe.)`
We'd better change vs_enterprise.exe to vs_installer.exe.